app/templates/components/component-tree-item.hbs

```diff
@@ -1,54 +1,111 @@
 <div
   style={{@item.style}}
   class="
-    component-tree-item
+    component-tree-item relative flex items-center mx-1 rounded
+    {{if @item.hasInstance "cursor-pointer" "cursor-default"}}
     {{if @item.isComponent "component-tree-item--component"}}
-    {{if @item.hasInstance "component-tree-item--has-instance"}}
-    {{if @item.isSelected "component-tree-item--selected"}}
-    {{if @item.isHighlighted "component-tree-item--highlighted"}}
```

Comment:
why is this removed?

Reply:
As far as I could tell that was being used for the hover state. See: https://github.com/emberjs/ember-inspector/pull/1129/files#diff-cbd2aa0d940eef1f44430e6ca030fbd9L8

Reply:
Here is the original PR: https://github.com/emberjs/ember-inspector/pull/984

Reply:
I see. Yeah that changed in Octane. It is now used to highlight the "currently previewing component". IMO the previous behavior is not that useful, since you can already see the parent/child relationship from the nesting.

There are two ways to "preview" a component, by hovering the rows in the component tree or by using the "live inspection" feature and hover over the DOM elements on the page:

![highlight](https://user-images.githubusercontent.com/55829/72568548-3af5cf00-386d-11ea-806b-772cf180a616.gif)

Using CSS hover state only solves the first part and I think we still need this for the latter.

Reply:
Oooooh. I see. I'll have to revert that.